hasura.planx.uk/metadata/tables.yaml

```diff
@@ -3027,21 +3027,24 @@
     - role: analyst
       permission:
         columns:
-          - action_colour
+          - live_flows
           - article_4s_enabled
-          - bops_submission_url
-          - favicon
+          - govpay_enabled_production
+          - govpay_enabled_staging
+          - is_trial
+          - planning_data_enabled
+          - power_automate_enabled_production
+          - power_automate_enabled_staging
+          - id
           - govnotify_personalisation
-          - govpay_enabled
+          - action_colour
+          - bops_submission_url_production
+          - bops_submission_url_staging
+          - favicon
```

Comment:
Something that took me way too long to work out and lead to me destroying, recreating, and rebasing my local env a bunch of times - if you destroy the only table where a role is used (`teams_summary` and `analyst`), then that role is also destroyed 🙃

Reply:
Not expected behavior !!